src/styles/utils/variables.scss

```diff
@@ -0,0 +1,7 @@
+$color-background: #cfee92;
+$color-day: #eee;
+$color-day-border: #000;
+$color-hover: pink;
+
```

Comment:
These variables are redundant but just for your info - it's better to name variable like `background-color`, `day-color,` `day-border-color`, `hover-color` as it sounds more clear

Reply:
Done